apps/src/templates/sectionAssessments/MultipleChoiceAnswerCell.jsx

```diff
@@ -45,8 +58,8 @@ class MultipleChoiceAnswerCell extends Component {
     }
 
     return (
-      <div style={styles.main}>
-        <div style={styles.text}>
+      <div style={{...styles.main, ...{...backgroundCSS}}}>
```

Comment:
I think this can be simplified to:
<div style={{...styles.main, ...backgroundCSS}}>